### PR TITLE
Fix error handling

### PR DESCRIPTION
**Pull request by krishung5**

### CODE REVIEW

File: src/openvino.cc [hunk 1]
```diff
@@ -227,7 +227,7 @@ ModelState::ParseParameters()
 {
   triton::common::TritonJson::Value params;
   bool status = model_config_.Find("parameters", &params);
-  if (status) {
+  if (status && !params.IsEmpty()) {
```
Comment (krishung5) [inline src/openvino.cc:230]: Model configuration will still have `parameters` field with empty content even there are no parameters set in the config.

**Comment by krishung5** [comment]
The previous logic should be fixed as it is possible that `parameters` field has some parameters but not all of them. We should not return error when one of the parameters is not found.

File: src/openvino_utils.cc [hunk 2]
```diff
@@ -330,10 +330,9 @@ ReadParameter(
     std::string* param)
 {
   triton::common::TritonJson::Value value;
-  RETURN_ERROR_IF_FALSE(
-      params.Find(key.c_str(), &value), TRITONSERVER_ERROR_INVALID_ARG,
-      std::string("model configuration is missing the parameter ") + key);
-  RETURN_IF_ERROR(value.MemberAsString("string_value", param));
+  if (params.Find(key.c_str(), &value)) {
+    RETURN_IF_ERROR(value.MemberAsString("string_value", param));
+  }
```
Comment (GuanLuo) [inline src/openvino_utils.cc:335]: I think we should return error, or some way to inform the caller of this function that the parameter is not found, then it is up to the caller to decide whether the parameter is optional.

Comment (krishung5) [reply src/openvino_utils.cc]: Changed to making the caller log out message when the parameter is not found. We can modify how to handle it If we need to make some of the parameters required in the future.

**Review by GuanLuo — APPROVED** [review]

**Pull request merged**